src/web/mod.rs

```diff
@@ -350,18 +136,15 @@ fn match_version(
         .expect("error fetching releases for crate");
 
     if releases.is_empty() {
-        return Err(Nope::CrateNotFound);
+        return Err(AxumNope::CrateNotFound);
     }
 
     // version is an Option<&str> from router::Router::get, need to decode first.
     // Any encoding errors we treat as _any version_.
-    use iron::url::percent_encoding::percent_decode;
-    let req_version = input_version
-        .and_then(|v| percent_decode(v.as_bytes()).decode_utf8().ok())
-        .unwrap_or_else(|| "*".into());
+    let req_version = input_version.unwrap_or("*");
```

Comment:
the version is correctly decoded by axum when extracting the path param from the URI